src/EzPlatformEncoreBundle/bundle/Command/CompileAssetsCommand.php

```diff
@@ -0,0 +1,80 @@
+<?php
+
+/**
+ * @copyright Copyright (C) eZ Systems AS. All rights reserved.
+ * @license For full copyright and license information view LICENSE file distributed with this source code.
+ */
+declare(strict_types=1);
+
+namespace EzSystems\EzPlatformEncoreBundle\Command;
+
+use Symfony\Bundle\FrameworkBundle\Command\ContainerAwareCommand;
+use Symfony\Component\Console\Input\InputInterface;
+use Symfony\Component\Console\Input\InputOption;
+use Symfony\Component\Console\Output\OutputInterface;
+use Symfony\Component\Process\Process;
+
+class CompileAssetsCommand extends ContainerAwareCommand
+{
+    public const COMMAND_NAME = 'ezplatform:encore:compile';
+
+    protected function configure(): void
+    {
+        $this
+            ->setName(self::COMMAND_NAME)
+            ->setDescription('Compiles all assets using WebPack Encore')
+            ->addOption(
+                'timeout',
+                't',
+                InputOption::VALUE_OPTIONAL,
+                'Timeout in seconds',
+                300
+            )
+        ;
+    }
+
+    protected function execute(InputInterface $input, OutputInterface $output): void
+    {
+        $timeout = $input->getOption('timeout');
+        $env = $input->getOption('env');
+
+        $output->writeln(sprintf('Compiling all <comment>%s</comment> assets.', $env));
+        $output->writeln('');
+
+        $encoreEnv = $env === 'prod' ? 'prod' : 'dev';
+        $yarnEncoreCommand = "yarn encore {$encoreEnv}";
```

Comment:
I wish we could detect if yarn is missing and fallback to npm if not, but besides that LGTM

Reply:
I suppose it can be done using `\Symfony\Component\Process\ExecutableFinder` :) But I will leave it to someone to come up with an improvement. `yarn` is required for now because we run `yarn install` from composer scripts. It could be also fixed to detect npm/yarn.

Reply:
ok 👍